### ISSUE TITLE
Provide options to only export png/svg/html instead of always all 3

### ISSUE THREAD

**Issue by andrew-leonard** [d3-node/d3node-output#11]
Has there been any thought into providing options to allow users to determine which output type they want instead of always providing png, svg & html? 

I have a use case where I only want the png output.

**Comment by andrew-leonard** [comment]
Happy to make a pull request and do this myself.

**Comment by bradoyler** [comment]
Sure. I’d be happy to merge it. :)

**Comment by austinrutledge0** [comment]
Did this ever get done?

**Comment by bradoyler** [comment]
no, but I'd prefer we don't change the default behavior, but if we added the options: `{ skipHtml, skipSvg }` that would be a simple change @ https://github.com/d3-node/d3node-output/blob/master/index.js#L49
